src/rfac/rfintindl.c

```
@@ -0,0 +1,150 @@
+      i++;
+    }
+
+    /* postscript scale factor */
+    if(command_line[i] == '*')
+    {
+      i++;
+      if(scale < 0. )
+      {
+        CLEED_SSCANF(command_line+i, fmt_buffer, &scale);
+      }
+      else CLEED_SSCANF(command_line+i, fmt_buffer, &scale);
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)